src/cmdline.rs

```diff
@@ -119,6 +121,13 @@ pub fn parse_args() -> Result<Config> {
                         .help("Embed an Ignition config from a file")
                         .takes_value(true),
                 )
+                .arg(
+                    Arg::with_name("ignition-hash")
+                        .long("ignition-hash")
+                        .value_name("type-value")
+                        .help("Digest of the Ignition config file (valid types: sha512)")
```

Comment:
Oh, I didn't know this was a concern. For now, I dropped the parenthetical text. Do you have a longer term plan for where to keep more verbose docs?

Reply:
[It is](https://github.com/coreos/coreos-installer/blob/b066a23993ab255756b3e2403c3280ea89d9b387/src/cmdline.rs#L72-L73), for readability.  Current candidates for verbose docs are the README and the FCOS install documentation, neither of which seems like a great solution.

The help text still exceeds 80 columns because `<type-value>` pushes the description column to the right.

Reply:
Ack, trimmed some more, it should be at 78 columns now.